components/GridView.vue

```diff
@@ -167,19 +160,57 @@ onUnmounted(() => {
           @moved="clearSelection"
           @resize="clearSelection"
           @resized="clearSelection"
+          drag-allow-from=".cursor-move"
+          drag-ignore-from=".z-10"
         >
+          <!-- This will serve as a handle to drag the widget when enabled -->
+          <div
+            v-if="editMode"
+            class="absolute left-0 top-0 z-20 cursor-move rounded-[10px] bg-white"
+          >
+            <lukso-icon
+              name="hand-right-outline"
+              size="small"
+              class="m-1"
+            ></lukso-icon>
+          </div>
           <GridWidget :widget="item" />
         </GridItem>
       </GridLayout>
     </div>
     <!-- This configuration tools are just temporal until we have the proper ones -->
-    <div class="fixed bottom-0 right-0 m-2">
+    <div class="fixed bottom-0 right-0 m-2 flex flex-col">
+      <lukso-button
+        v-if="editMode"
+        size="small"
+        type="button"
+        variant="secondary"
+        is-icon
+        disabled="true"
+      >
+        <lukso-icon name="plus" size="medium" class="mx-1"></lukso-icon>
+      </lukso-button>
       <lukso-button
         size="small"
         type="button"
         variant="secondary"
+        is-icon
+        @click="toggleEditMode()"
```

Comment:
```suggestion
        @click="toggleEditMode"
```

We prefix click functions with `handle`. Brackets are not necessary.